src/components/Playground.tsx

```diff
@@ -418,7 +418,13 @@ class Playground extends React.Component<IPlaygroundProps, PlaygroundState> {
 
   private processArrayOutput = (output: InterpreterOutput[]) => {
     const editorOutput = output[0];
-    if (editorOutput && editorOutput.type === 'result' && editorOutput.value instanceof Array) {
+    if (
+      editorOutput &&
+      editorOutput.type === 'result' &&
+      editorOutput.value instanceof Array &&
+      editorOutput.value[0] instanceof Array &&
+      typeof editorOutput.value[0][0] === 'string'
```

Comment:
This is not obvious. Can you add a comment what problem this fixes and how it fixes it?

Reply:
Since the stepper input is of type Array<[string, string, string]>, the above checks the editor output value for the following:
1. Is an array
2. First element is an array
3. First element of that array is a string

Will be adding a commit that includes this comment in the code!